conjure-go-client/httpclient/client_params.go

```diff
@@ -470,6 +471,19 @@ func WithBasicAuth(username, password string) ClientParam {
 	}))
 }
 
+// WithBalancedURIScoring adds middleware that prioritizes sending requests to URIs with the fewest in-flight requests
+// and least recent errors.
+func WithBalancedURIScoring() ClientParam {
```

Comment:
Given that balanced scoring is the default setting, should we instead expose a client param to use the random scorer (or similar) instead? Also it's helpful to add what the default value is to the godoc comment. For example, this currently does nothing since balanced scoring is already the default.

Reply:
Oh my bad I meant to make the random scorer the default and expose this for consumers who want the balanced scoring middleware

Reply:
I feel like this could be the default option and those that want to opt out could choose another strategy, but this seems fine for now and we can re-asses down the road if we add more strategies.